.gitignore

```diff
@@ -103,6 +104,8 @@ venv.bak/
 # mypy
 .mypy_cache/
 
+# vscode
```

Comment:
Can you put editor-specific gitignore rules in your global gitignore?

Reply:
This isn't global, and you should absolutely ignore any/all editor related files that are generated.

Reply:
From the docs on git (`https://git-scm.com/docs/gitignore`):
```
Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig. Its default value is $XDG_CONFIG_HOME/git/ignore. If $XDG_CONFIG_HOME is either not set or empty, $HOME/.config/git/ignore is used instead.
```

I agree that editor-related files should be ignored, just that this isn't the right place for it by convention.

Reply:
Good idea, still keeping this one line in here though, in this massively tiny file